hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMock.java

```diff
@@ -58,34 +54,86 @@ public void setObjectMapper(ObjectMapper objectMapper) {
     }
 
     public void addTopic(String topicName) {
-        addTopic(topicName, RESPONSE_CODE.CREATED);
+        addTopic(topicName, HttpStatus.SC_CREATED);
+    }
+
+    public void addAvroTopic(String topicName) {
+        addAvroTopic(topicName, HttpStatus.SC_CREATED);
     }
 
-    public void addTopic(String topicName, RESPONSE_CODE statusCode) {
-        addTopic(topicName, aResponse().withStatus(statusCode.code));
+    public void addTopic(String topicName, int statusCode) {
+        addTopic(topicName, false, aResponse().withStatus(statusCode));
     }
 
-    public void addTopic(String topicName, ResponseDefinitionBuilder responseDefinitionBuilder) {
-        wireMockServer.stubFor(get(urlPathMatching("/topics/" + topicName))
-                .willReturn(responseDefinitionBuilder
-                        .withHeader("Content-Type", "application/json")
-                        .withHeader("Hermes-Message-Id", UUID.randomUUID().toString())
-                )
-        );
+    public void addAvroTopic(String topicName, int statusCode) {
+        addTopic(topicName, true, aResponse().withStatus(statusCode));
+    }
+
+    public void addTopic(String topicName, Boolean isAvro, ResponseDefinitionBuilder responseDefinitionBuilder) {
```

Comment:
Should this method be public?
Maybe instead of passing `isAvro` flag, we could pass `contentType`?

Reply:
1. Should this method be public?
- Why not? If you want to create a detailed response, please, specify is it in avro format or not. Well we can add `public void addTopic(String topicName, ResponseDefinitionBuilder responseDefinitionBuilder)` for this too.
2. Maybe instead of passing isAvro flag, we could pass contentType?
- Hermes supports only 2 format atm(json & avro), if user would like to specify custom content type method from previous answer can be used.

Reply:
1. Because now we expose Wiremock to user, but this may be fine.
2. I like `public void addTopic(String topicName, ResponseDefinitionBuilder responseDefinitionBuilder` better. Boolean as a method argument just screams to me "you really have got 2 methods"